Explorer/Assets/DCL/Utilities/NftUtils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace DCL.Utilities
+{
+    public class NftUtils
+    {
+        public static bool TryParseUrn(string urn, out string contractAddress, out string tokenId)
```

Comment:
I propose to use `URN` type everywhere to be consistent

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/398/commits/543f0b6b95b2991868dde6bd7715765a8dd7e77a